### ISSUE TITLE
Videos converted with Playdate Video Encoder web interface show up as a black screen

### ISSUE THREAD

**Issue by klundry** [hteumeuleu/playorama#6]
I've tried a couple different videos from different sources and the resulting file just gives me a black screen.

**Comment by hteumeuleu** [comment]
Hey there. Thanks for reporting this issue. Could you elaborate a bit more? 

1. Which version of Playorama do you use?
2. Did you sideload via USB or via Panic's website?
3. In Playorama, if you go to Menu > Log, is there any error listed?
4. Could you share any video you tried?

Thank you!

**Comment by klundry** [comment]
1. Playorama v 1.0.3
2. Sideloaded via USB
3. No log entries.
4. Here are two that I have tried. 
https://ia801500.us.archive.org/34/items/2022ltmmever/0005%20-%20Box%20Car%20Blues%20%28DVD%29.mp4
https://vod.newellijay.tv/w/pdGKzj9hDHbTmw8RNGnNyB

The audio plays but the video is black. However, if you watch for a little bit you'll occasional see a flicker of an actual frame of the video but it's justa split second here and there and then back to black.  It also only shows black on the preview in the converter web interface.

**Comment by klundry** [comment]
I did a little more investigation and it appears to be related to firefox. Same file selected when using chromium comes out looking as it should. But on firefox it comes out as a black screen.

**Comment by hteumeuleu** [comment]
Which version of Firefox do you use? I'm on Firefox 119 on macOS 13.6 and have no problem with both videos you shared.

**Comment by klundry** [comment]
I'm on 118 on Linux.

**Comment by klundry** [comment]
Is the encoder web interface using ffmpeg on the backend? I'd probably prefer just processing videos locally but I'm not sure how to get ffmpeg or another tool to encode it correctly. The Crankplayer script works but the dithering isn't as good as some of the options on the web encoder.

**Comment by hteumeuleu** [comment]
No, the web interface is purely hand made JavaScript. But I'd love to have a command line tool to encode `pdv` at some point. Not sure if it's possible to plug this into ffmpeg or if I'd rather straight away go to something in Node. 

Regarding your issue, do you have any error in your DevTools console in Firefox?
